packages/shared/styles/components/SfGallery.scss

```
@@ -16,19 +16,24 @@ $gallery__stage-width: 400px !default;
   display: flex;
   flex-direction: $gallery-flex-direction;
   position: relative;
+  @media (max-width: $mobile-max) {
```

```suggestion
  @media (min-width: $desktop-min) {
```
i'ts detail but for desktop please use min-width: $desktop-min becoue we use it in other components styles